src/functions/components/FunctionCard.tsx

```diff
@@ -0,0 +1,56 @@
+import React, {FC} from 'react'
+import {useParams, useHistory} from 'react-router-dom'
+
+// Components
+import {
+  AlignItems,
+  ComponentSize,
+  FlexDirection,
+  ResourceCard,
+  FlexBox,
+} from '@influxdata/clockface'
+import FunctionContextMenu from 'src/functions/components/FunctionContextMenu'
+import LastRunFunctionStatus from 'src/functions/components/LastRunFunctionStatus'
+
+interface Props {
+  id: string
+  name: string
+}
+
+const FunctionCard: FC<Props> = ({id, name}) => {
+  const {orgID} = useParams<{orgID: string}>()
```

Comment:
grab the org from redux using `const { org } = useSelector(getOrg)` here instead. then we have one definition of "what is the current org" for when we finally add an org selector in there